package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "dependencies": {
     "@bbc/aes31-adl-composer": "^1.1.0",
-    "@bbc/digital-paper-edit-storybook": "^1.27.0",
-    "@bbc/fcpx-xml-composer": "^1.1.0",
+    "@bbc/digital-paper-edit-storybook": "^1.28.0",
```

Comment:
Version 1.29 is on NPM - just checking that this is the one you're after! https://www.npmjs.com/package/@bbc/digital-paper-edit-storybook

Reply:
yeah I think it has 1.29 anyway cause of the ^ – but maybe I'll make it 1.29 anyway just so it's super clear!